src/eShop.AppHost/Extensions.cs

```diff
@@ -38,8 +38,10 @@ public static IDistributedApplicationBuilder AddOpenAI(this IDistributedApplicat
         IResourceBuilder<ProjectResource> webApp)
     {
         const string openAIName = "openai";
-        const string textEmbeddingModelName = "text-embedding-3-small";
-        const string chatModelName = "gpt-4o-mini";
+        const string textEmbeddingAzureDeployName = "text-embedding-3-small"; // unique identifier for specific deployment within Azure OpenAI resource
+        const string textEmbeddingModelName = textEmbeddingAzureDeployName; // actual OpenAI model used in deployment
+        const string chatModelAzureDeployName = "gpt-4o-mini";
+        const string chatModelName = chatModelAzureDeployName;
```

Comment:
I'd swap these and assume that deploymentName == modelName. In its current form, in case these differ, you'd be changing all of the values.

Reply:
Cheers!